src/components/footer/Footer.jsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import "./footer.css";
+import { Link } from "react-router-dom";
+
+import { FooterLinks } from "./footer.data";
+
+import { logo } from "../../assets";
+
+const Footer = () => {
+  const { categoryLinks, accountLinks, helpLinks, socialLinks } = FooterLinks;
+
+  return (
+    <footer className="basic-bg flex-row wrap items-start">
+      <div className="footer-item footer-links">
+        <h6 className="text-uppercase underline">Categories</h6>
+        <ul className="links">
+          {categoryLinks.map(({ id, linkName, url }, index) => (
+            <li key={id}>
+              <Link className="link" to={url}>
+                {linkName}
+              </Link>
+            </li>
+          ))}
+        </ul>
+      </div>
+
+      <div className="footer-item footer-links">
+        <h6 className="text-uppercase underline">Account</h6>
+        <ul className="links">
+          {accountLinks.map(({ id, linkName, url }, index) => (
+            <li key={id}>
+              <Link className="link" to={url}>
+                {linkName}
+              </Link>
+            </li>
+          ))}
```

Comment:
Good that you have mapped the repeating structure and also de-structured the object. 
Suggestion : I think index is not required here.

Reply:
Thanks! 
I'll need index later. That's why I have added it here.